src/components/Card.tsx

```diff
@@ -82,7 +82,9 @@ export const Card = React.memo((props: CardProps) => {
             </View>
         );
     }
-});
+};
+
+export const MemoizedCard = React.memo(Card);
```

Comment:
Where do we use the plain (not memoized) `Card`? If we don't use it, can't we just use the name `Card` for the memoized version?